client/job.lua

```diff
@@ -408,15 +408,20 @@ RegisterNetEvent('police:client:TakeOutImpound', function(data)
         TakeOutImpound(vehicle)
     end
 end)
-
+function TakeOutVehicle(vehicleInfo)
+    local coords = Config.Locations["vehicle"][currentGarage]
+    QBCore.Functions.SpawnVehicle(vehicleInfo, function(veh)
+        SetVehicleNumberPlateText(veh, "AMBU"..tostring(math.random(1000, 9999)))
```

Comment:
Does this not needed to be POLICE instead of AMBU?

Reply:
Actually, it needs to be "PLZI" for compatibility. Unless the change is made here and then in [qb-inventory\server\main.lua line 669](https://github.com/qbcore-framework/qb-inventory/blob/eae5953297fb7fa8511ca65c279fcf21d4f3db86/server/main.lua#L669)